src/help/guides/getting-started.md

```diff
@@ -6,99 +6,34 @@ lang: en-US
 # Getting started
 
 ## Installation
-**Paperback** is currently _not_ on **TestFlight** and _requires_ the usage of **AltStore** in order to install onto your iPhone or iPad. In the future, TestFlight will be used when the lead developer obtains an [Apple Developer Account](https://developer.apple.com/programs/).
-
-::: tip Video tutorial
-If you prefer using a video tutorial, Artesians made a beautiful one. You will find it [here](https://www.youtube.com/watch?v=n1KRwsxNiWY) for Windows and [here](https://www.youtube.com/watch?v=CjPjsF4yJ0M) for macOS.
-:::
-
-### Prerequisites
-1. On your iPhone or iPad, download the app <Download text="from here"/>. The app will download as **`Paperback.ipa`**.
-1. Download [AltServer](https://altstore.io/) for your operating system.
-
-::: tip Pre-releases
-On [GitHub](https://github.com/Paperback-iOS/app/releases), there are also pre-releases. These are only available for Patrons. If you are not a Patron, make sure to install the **Stable** builds.
-If you are interested in testing new features early, consider becoming a Patron.
-:::
+Currently an early version of the app is available on the App Store! [Click here](https://apps.apple.com/na/app/paperback-manga-reader/id1519509781) to download a copy.
+If you are a patreon, you will recieve an email to your patreon address regarding information about accessing Paperback through TestFlight. The TestFlight version of the application contains many additional beta features which can be used before it becomes available on the App Store.
 
 ::: warning Warning
 This app requires iOS 13.4+ or iPadOS 13.4+. If you have an older version, the app will CRASH. Paperback is NOT designed for older iOS versions.
 :::
 
 :::: el-tabs
-::: el-tab-pane label="macOS 10.14+"
-### AltStore
+::: el-tab-pane label="Public Paperback version"
```

Comment:
A really personal opinion: wouldn't just "Public version" be better? I think "Public Paperback version" is really long.
```suggestion
::: el-tab-pane label="Public version"
```